ttyd/umbrel-app.yml

```diff
@@ -0,0 +1,23 @@
+manifestVersion: 1
+id: ttyd
+category: DEVELOPMENT
+name: ttyd
+version: "1.7.3"
+tagline: Share your terminal over the web
+description: >-
+  ttyd is a simple command-line tool for sharing terminal over the web.
+releaseNotes: >-
+  Sixel support is off by default now, use -t enableSixel=true to enable it.
+developer: tsl0922
+website: https://tsl0922.github.io/ttyd/
+repo: https://github.com/tsl0922/ttyd
+support: https://github.com/tsl0922/ttyd/issues
+port: 7681
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+dependencies: []
+path: ""
+defaultUsername: ""
+defaultPassword: ""
```

Comment:
You can add `submitter: <you!>` and `submission: <link to this PR>` at the bottom here.